src/documentation/templates/payment-process.html

```diff
@@ -38,10 +40,10 @@ <h3 class="m-b-24">Proceso de pago</h3>
                 <a href="#" class="dp-edit-form" title="Editar formulario"
                   ><span class="ms-icon icon-edit"></span
                 ></a>
-                <div class="dp-accordion-panel">
+                <div class="dp-accordion-panel" style="display: block">
```

Comment:
Is adding this inline style fine?

Reply:
It's just to expand the accordion. Does not affect the place where it is going to be used